app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```

Comment:
[Q]
Could you please use I18.n to text?
Currently, Our system support multi-language. This is good change to practice that

Reply:
[A] OK I'll try.